### ISSUE TITLE
Need to install windows sdk 10.0.17763.0

### ISSUE THREAD

**Issue by Laeeth** [dlang-cpp-interop/stl-containers#20]
Or change the solution.

https://ci.appveyor.com/project/LaeethIsharc/stl-containers/builds/24694207/job/q0nslps2kosk09y7

Project "C:\projects\stl-containers\stl-containers.sln" (1) is building "C:\projects\stl-containers\stl-containers.vcxproj" (2) on node 1 (default targets).
C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Platforms\x64\PlatformToolsets\v140\Toolset.targets(36,5): error MSB8036: The Windows SDK version 10.0.17763.0 was not found. Install the required version of Windows SDK or change the SDK version in the project property pages or by right-clicking the solution and selecting "Retarget solution". [C:\projects\stl-containers\stl-containers.vcxproj]
Done Building Project "C:\projects\stl-containers\stl-containers.vcxproj" (default targets) -- FAILED.
Done Building Project "C:\projects\stl-containers\stl-containers.sln" (default targets) -- FAILED.
Build FAILED.

**Comment by rmanthorpe** [comment]
This is a visual studio problem - it makes solutions target a specific Windows SDK version even though it usually doesn't matter. I manged to fix this for my solutions by adding this:

``` xml
  <PropertyGroup Condition="'$(DefaultPlatformToolset)' == 'v141'" Label="Globals">
    <LatestTargetPlatformVersion>$([Microsoft.Build.Utilities.ToolLocationHelper]::GetLatestSDKTargetPlatformVersion('Windows', '10.0'))</LatestTargetPlatformVersion>
    <WindowsTargetPlatformVersion>$(LatestTargetPlatformVersion)</WindowsTargetPlatformVersion>
    <TargetPlatformVersion>$(WindowsTargetPlatformVersion)</TargetPlatformVersion>
  </PropertyGroup>
```

**Comment by TurkeyMan** [comment]
Yeah, a solution like this is usually possible, and if it doesn't work out with just one hard sln, I usually use premake to generate msbuild projects for the select versions. I think premake is the best build tool for use with D, it supports mixed projects properly, and the msbuild support is better than anything else.
